packages/@webex/internal-plugin-metrics/src/call-diagnostic/call-diagnostic-metrics-latencies.ts

```diff
@@ -93,31 +95,31 @@ export default class CallDiagnosticLatencies extends WebexPlugin {
    * Store precomputed latency value
    * @param key - key
    * @param value - value
-   * @param overwrite - overwrite existing value or add it
+   * @param accumulate - when it is true, it overwrites existing value with sum of the current value and the new measurement otherwise just store the new measurement
    * @throws
    * @returns
    */
-  public saveLatency(key: PreComputedLatencies, value: number, overwrite = true) {
-    const existingValue = overwrite ? 0 : this.precomputedLatencies.get(key) || 0;
+  public saveLatency(key: PreComputedLatencies, value: number, accumulate = false) {
+    const existingValue = accumulate ? this.precomputedLatencies.get(key) || 0 : 0;
     this.precomputedLatencies.set(key, value + existingValue);
   }
 
   /**
    * Measure latency for a request
-   * @param key - key
    * @param callback - callback for which you would like to measure latency
-   * @param overwrite - overwite existing value or add to it
+   * @param key - key
+   * @param accumulate - when it is true, it overwrites existing value with sum of the current value and the new measurement otherwise just store the new measurement
    * @returns
    */
   public measureLatency(
-    callback: () => Promise<any>,
+    callback: () => Promise<unknown>,
     key: PreComputedLatencies,
-    overwrite = false
+    accumulate = false
```

Comment:
Are you planning to change definition in Cantina too, I think we should

Reply:
Yes, I already have the PR for that
https://sqbu-github.cisco.com/WebExSquared/webex-web-client/pull/5653

Reply:
We don't have accumulateLatency anymore right?